test/unit_Polygon.jl

```diff
@@ -396,4 +400,4 @@ end
 # default Float64 constructors
 @test HPolygon() isa HPolygon{Float64}
 @test HPolygonOpt() isa HPolygonOpt{Float64}
-@test VPolygon() isa VPolygon{Float64}
+@test VPolygon() isa VPolygon{Float64}
```

Comment:
Why remove the empy line here?

Reply:
(it was unintentional)

Thanks for the review. I'll update those changes and merge when CI passes.